setup.qmd

```diff
@@ -115,36 +115,31 @@ See [brew.sh](https://brew.sh/) for installation instructions.
 To install Git using Homebrew, run the following in the Terminal: `brew install git`
 :::
 
-## GitHub {#github}
+## GitLab {#gitlab}
 
-Register a GitHub account at [github.com](https://github.com/).
+Register a GitLab account at [gitlab.com](https://gitlab.com/).
 
 I highly recommend the book [Happy Git with R](https://happygitwithr.com/) for advice on setting up and using Git with R [@bryan].
 
-## Check link between R and GitHub {#check-link-between-r-and-github}
+## Check link between R and GitLab {#check-link-between-r-and-gitlab}
 
 Configure your `user.name` and `user.email` for git in RStudio with:
 
 ```{r}
 #| eval: false
 usethis::use_git_config(
   user.name = "Jane Doe",          # actual first and last name
-  user.email = "jane@example.com"  # email associated with GitHub account
+  user.email = "jane@example.com"  # email associated with GitLab account
   )
 ```
 
-Communicating with GitHub requires authentication with your GitHub account.
+Communicating with GitLab requires authentication with your GitLAb account.
 This is achieved by creating and securely storing a Personal Access Token (PAT).
-The [Managing Git(Hub) Credentials](https://usethis.r-lib.org/articles/articles/git-credentials.html) vignette in the **`usethis`** [@usethis] package has details on using PATs, but for now the following should suffice.
+Although focused more on GitHub, the [Managing Git(Hub) Credentials](https://usethis.r-lib.org/articles/articles/git-credentials.html) vignette in the **`usethis`** [@usethis] package has details on using PATs, but for now the following should suffice.
 
-Create a PAT with:
+Create a PAT with using these instructions: https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token
```

Comment:
Should we give suggestions on scopes and expiry?

Reply:
I think this is reasonable: api, read_api, read_user, create_runner, read_repository, write_repository

Reply:
great thanks for this.